test/unit/server-statefulset.bats

```diff
@@ -347,7 +363,7 @@ load _helpers
   local object=$(helm template \
       --show-only templates/server-statefulset.yaml  \
       --set 'server.standalone.enabled=true' \
-      --set 'server.extraVolumes[0].type=configMap' \
+      --set 'server.extraVolumes[0].type=secret' \
```

Comment:
This change and the other 3 like it are intentional as I believe they're duplicate tests and this was the original intention.

If my PR gets rejected, fixing the tests might still be worthwhile.